lib/persisted-model.js

```diff
@@ -619,6 +668,20 @@ module.exports = function(registry) {
       rest: {after: convertNullToNotFoundError}
     });
 
+    setRemoting(PersistedModel, 'replaceById', {
+      description: 'Replace attributes for a model instance and persist it into the data source.',
+      accessType: 'WRITE',
+      accepts: [
+        { arg: 'id', type: 'any', description: 'Model id', required: true,
+          http: { source: 'put' }},
+          { arg: 'data', type: 'object', http: { source: 'body' }, description:
+        'An object of model property name/value pairs' },
```

Comment:
Should the description say the same as in `replaceOrCreate`, i.e. `Model instance data`?